packages/astro/src/core/build/plugins/plugin-noop.ts

```diff
@@ -17,7 +17,7 @@ export function pluginNoop(): vite.Plugin {
 		},
 		load(id) {
 			if(id === RESOLVED_NOOP_MODULE_ID) {
-				return '';
+				return 'export const noop = {};';
```

Comment:
If this is confusing just read the code below this, the chunk gets deleted at the end.